README.md

```diff
@@ -170,6 +173,60 @@ $ grunt bump:git
 >> Committed as "Release v1.0.1-ge96c"
 >> Tagged as "v1.0.1-ge96c"
 >> Pushed to origin
+
+$ grunt bump:prepatch
+>> Version bumped to 1.0.2-rc.0
+>> Committed as "Release v1.0.2-rc.0"
+>> Tagged as "v1.0.2-rc.0"
+>> Pushed to origin
+
+$ grunt bump:prerelease
```

Comment:
You missed:

> A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version

Meaning everything after the hyphen is the identifier. To have multiple identifiers, you separate them with dots.

As for the current current behavior of going from `1.0.0` to `1.0.0-1` as the next prerelease, that is a bug that existed in semver which has since been fixed. Purely upgrading semver with no other changes would make it go from `1.0.0` to `1.0.1-0`.